README.md

```diff
@@ -502,6 +502,7 @@ Below are some examples
 * [CoffeeScript](https://www.npmjs.com/package/coffee-script): `--compiler coffee:coffee-script/register`
 * [TypeScript](https://www.npmjs.com/package/ts-node): `--compiler ts:ts-node/register`
 * [Pogo](https://www.npmjs.com/package/pogo): `--compiler pogo:pogo`
+* [LiveScript](http://livescript.net): `--compiler ls:LiveScript`
```

Comment:
livescript 1.4.0 [changed](http://livescript.net/#changelog) its npm package name to all lowercase. I'd suggest we use that. 

I'm also not entirely sure about adding another transpiler example. These aren't here to promote anyone in particular but just to offer an example. I'd actually rather just simplify this down to one example..